src/pages/iou/request/step/IOURequestStepCategory.tsx

```diff
@@ -109,24 +112,52 @@ function IOURequestStepCategory({
         const categorySearchText = category.searchText ?? '';
         const isSelectedCategory = categorySearchText === transactionCategory;
         const updatedCategory = isSelectedCategory ? '' : categorySearchText;
+        const categoryTaxCode = CategoryUtils.getCategoryDefaultTaxRate(policy?.rules?.expenseRules ?? [], updatedCategory, policy?.taxRates?.defaultExternalID);
```

Comment:
I pass the default tax rate here because all the categories on the WS category setting is default to 0%.

<img width="381" alt="Screenshot 2024-11-30 at 16 34 31" src="https://github.com/user-attachments/assets/1550b3b8-3a52-46c8-a6d8-01c5c90b1d32">

With default:

Selecting another category in which we never set the tax rate before will update the tax rate to the default value, 0% in my case.

https://github.com/user-attachments/assets/989a642e-055a-49a4-98de-edecc035b9c1


Without default:

Selecting another category in which we never set the tax rate before won't update the tax rate.

https://github.com/user-attachments/assets/dcea0a60-50e2-4fc0-a19b-ab82e912aabf

@MonilBhavsar

Reply:
> on the WS category setting is default to 0%.

It should default to workspace default tax rate on settings in Tax page. Your default tax setting should be 0% in this case